options.yaml

```
@@ -434,12 +434,20 @@
     Basic tells if the reward item is any progress item. Advanced tells if the trial is WotH, Barren, or potentially required.
     Direct tells the item on the trial directly.
   ui: option_song_hints
-- name: Fix BiT crashes
-  command: fix-bit-crashes
-  type: boolean
-  default: false
-  help: If enabled, does not load some arcs in BiT when all file slots are filled to make all areas accessible in BiT
-  ui: option_fix_bit_crashes
+- name: BiT Changes
+  command: bit-patches
+  type: singlechoice
+  bits: 2
+  choices:
+    - Disable BiT
+    - Vanilla
+    - Fix BiT Crashes
+  default: Vanilla
+  help: "Changes how the Back in Time (BiT) glitch works.
+        Disable BiT: Forces the title screen to reload fully - making all BiT tricks impossible.
```

```suggestion
        Disable BiT: Makes it impossible to activate BiT.
```
`Forces the title screen to reload fully` is not accurate at all, the title screen loads fully and properly in any case, the difference is really if link respects the params of the entrance, that are supposed to trigger an infinite cutscene